tests/test_port_dpb_system.py

```diff
@@ -248,25 +242,380 @@ def test_port_breakout_with_acl(self, dvs):
 
         # Verify port is removed from ACL table
         self.dvs_acl.verify_acl_table_count(1)
-
-        #TBD: Uncomment this, or explain why Ethernet0 is being added back to ACL table
-        # Also, string "None" is being added as port to ACL port list after the breakout
-        #self.dvs_acl.verify_acl_group_num(0)
+        self.dvs_acl.verify_acl_group_num(0)
 
         # Verify child ports are created.
         self.verify_only_ports_exist(dvs, ["Ethernet0", "Ethernet1", "Ethernet2", "Ethernet3"])
 
-        # Enable below snippet after fixing the above issues
-        '''
         # Move back to 1x100G[40G] mode and verify current mode
         dvs.change_port_breakout_mode("Ethernet0", "1x100G[40G]", "-f")
         dpb.verify_port_breakout_mode(dvs, "Ethernet0", "1x100G[40G]")
-        '''
 
         # Remove ACL table and verify the same
         self.dvs_acl.remove_acl_table("test")
         self.dvs_acl.verify_acl_table_count(0)
 
+    def test_cli_command_with_hyphen_f_option(self, dvs):
```

Comment:
We can add a reference to the test plan on this PR, so that it is easy to refer for completeness of test coverage.

Reply:
I do NOT have a formal plan, But I can reference to your document. But also note that, there no correlation between your document and the test case, so that will confuse the reader even more. Instead, I can copy paste the list of cases I documented in Jira ticket.

Reply:
Okies so test plan is in your Jira Ticket, No probs, you can paste here or in the conversation of this PR.

Reply:
Done.

Reply:
Done.